tests/serialization/test_dag_serialization.py

```diff
@@ -1233,6 +1233,66 @@ def test_deps_sorted(self):
             'airflow.ti_deps.deps.trigger_rule_dep.TriggerRuleDep',
         ]
 
+    def test_derived_dag_deps(self):
+        """
+        Tests DAG dependency detection, including derived classes
+        """
+        from airflow.operators.dummy import DummyOperator
+        from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+        from airflow.sensors.external_task import ExternalTaskSensor
+
+        class DerivedSensor(ExternalTaskSensor):
+            template_fields = ['external_dag_id', 'run_id']
+
+            def __init__(self, *, external_dag_id: str, run_id: str = None, **kwargs):
+                super().__init__(
+                    external_dag_id=external_dag_id, execution_delta=None, execution_date_fn=None, **kwargs
+                )
+                self.run_id = run_id
```

Comment:
```suggestion
        class DerivedSensor(ExternalTaskSensor):
            pass
```

would this not accoplish the same thing?  is there a reason we need to add the other stuff for this test?